.github/workflows/ci.yml

```diff
@@ -25,6 +25,11 @@ jobs:
       - name: Run tests
         run: poetry run pytest --cov-report=xml
 
+      - name: Run mypy
+        # run even if tests fail
+        if: always()
```

Comment:
This is here to ensure that both, tests and mypy are run, regardless of the result of each of them, right?

Reply:
Though I've realised we should probably do it the other way round, because now the code coverage won't be uploaded unless `mypy` succeeds